components/places/android/src/main/java/org/mozilla/places/PlacesConnection.kt

```diff
@@ -22,13 +23,13 @@ import java.nio.ByteOrder
  *  database. If omitted, data will be stored in plaintext.
  */
 class PlacesConnection(path: String, encryption_key: String? = null) : PlacesAPI, AutoCloseable {
-    private var db: RawPlacesConnection?
+    private var handle: AtomicLong = AtomicLong(0)
```

Comment:
Worth adding a `typealias` for Kotlin, too?

Reply:
For AtomicLong? I could see it for `Long`, which is actually what we pass into the FFI types, but for AtomicLong I don't fully see the benefit, since it's an implementation detail internal to that class.